### PR TITLE
Refactor/move query execution

### PR DESCRIPTION
**Pull request by x3medima17**
### Description of the Change
Gets `QueryExecution` out from old model

### Benefits
One less component in old model

### Possible Drawbacks 
None

### CODE REVIEW

File: irohad/execution/query_execution.hpp [hunk 1]
```diff
@@ -0,0 +1,135 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
```
Comment (l4l) [inline irohad/execution/query_execution.hpp]: so old(

File: irohad/execution/query_execution.hpp [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+  namespace interface {
+    class QueryResponse;
+    class Query;
+  }  // namespace interface
+}  // namespace shared_model
+
+namespace iroha {
+
+  /**
+   * Converting business objects to protobuf and vice versa
```
Comment (l4l) [inline irohad/execution/query_execution.hpp]: business -> model

File: irohad/execution/query_execution.hpp [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+  class QueryProcessingFactory {
+    using QueryResponseBuilder =
+        shared_model::proto::TemplateQueryResponseBuilder<0>;
+
+    using QueryResponseBuilderDone =
+        shared_model::proto::TemplateQueryResponseBuilder<1>;
+
+   public:
+    /**
+     * Execute and validate query.
```
Comment (grimadas) [inline irohad/execution/query_execution.hpp]: If this method executes and validates, probably the naming is wrong?

**Comment by Alexey-N-Chernyshov** [comment]
Please fix CMakeLists in `torii/processor/CMakeLists.txt` as well. There is old model.

**Pull request merged**